logstash.conf.m4

```diff
@@ -32,13 +34,13 @@ filter {
    code => "event['timestamp'] /= 1_000_000"
```

Comment:
I agree with your argument for changing `timestamp` to `timestamp_ms`, so please put it back